infrastructure/docker/celo/README.md

```diff
@@ -0,0 +1,19 @@
+# Celo client Docker image for local development
```

Comment:
Can we add a `.gitignore` file containing `data` dir path?

Reply:
I wasn't sure about it given that the developer may point to any directory when starting the container. However, given that the README mentioned `$(pwd)/data`, ignoring it by default is good for developer's convenience when running everything as README suggests.